actionview/lib/action_view/railtie.rb

```diff
@@ -51,6 +51,14 @@ class Railtie < Rails::Engine # :nodoc:
       end
     end
 
+    initializer "action_view.setup_etag_with_template_digest" do |app|
```

Comment:
@rafaelfranca I committed this and the next commit using your name and email address, since you did all the work.  Please check the commit messages and re-word them if you feel that they don't capture the intent of the commit.